talentsearch/resources/js/components/request/CreateRequest.tsx

```diff
@@ -0,0 +1,303 @@
+            <Input
+              id="email"
+              type="email"
+              name="email"
+              label={intl.formatMessage({
+                defaultMessage: "Government e-mail",
+                description:
+                  "Label for government email input in the request form",
+              })}
+              placeholder={intl.formatMessage({
+                defaultMessage: "example@canada.ca...",
+                description:
+                  "Placeholder for government email input in the request form",
+              })}
+              rules={{ required: errorMessages.required }}
+            />
+          </div>
+          <div
+            style={{ paddingLeft: "1rem" }}
+            data-h2-flex-item="b(1of1) m(1of2)"
+          >
+            <Select
+              id="department"
+              name="department"
+              label={intl.formatMessage({
+                defaultMessage: "Department / Hiring Organization",
+                description:
+                  "Label for department select input in the request form",
+              })}
+              nullSelection={intl.formatMessage({
+                defaultMessage: "Select a department...",
+                description:
+                  "Null selection for department select input in the request form.",
+              })}
+              options={departmentOptions}
+              rules={{ required: errorMessages.required }}
+            />
+            <Input
+              id="jobTitle"
+              type="text"
+              name="jobTitle"
+              label={intl.formatMessage({
+                defaultMessage: "What is the job title for this position?",
+                description: "Label for job title input in the request form",
+              })}
+              placeholder={intl.formatMessage({
+                defaultMessage: "Developer...",
+                description:
+                  "Placeholder for job title input in the request form.",
+              })}
+              rules={{ required: errorMessages.required }}
+            />
+          </div>
+          <div data-h2-flex-item="b(1of1)">
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "In this field please include any additional details and qualifications you are seeking from the candidates such as: programming languages, certifications, knowledge, or a specific work location.",
+                description:
+                  "Blurb before additional comments textarea in the request form.",
+              })}
+            </p>
+            <TextArea
+              id="additionalComments"
+              name="additionalComments"
+              label={intl.formatMessage({
+                defaultMessage: "Additional Comments",
+                description:
+                  "Label for additional comments textarea in the request form.",
+              })}
+            />
+          </div>
+          <SummaryOfFilters
```

Comment:
Wouldn't it be better to leave this as empty, pass fake data for the storybook stories, and then fill these with the real data once that is available?

Reply:
Ya good point

Reply:
I think you forgot to remove the values for this.

Reply:
Which values are you referring too?

Reply:
I believe I have removed all the fake data.